src/psbtv2.test.js

```diff
@@ -1052,14 +1055,73 @@ describe("PsbtV2.addPartialSig", () => {
     psbt.addPartialSig(0, Buffer.from([0x00]), Buffer.from([0x00]));
     psbt.addPartialSig(0, Buffer.from([0x01]), Buffer.from([0x01]));
     psbt.addPartialSig(1, Buffer.from([0x02]), Buffer.from([0x02]));
-    expect(psbt.PSBT_IN_PARTIAL_SIG[0]).toEqual(
+    expect(psbt.PSBT_IN_PARTIAL_SIG).toEqual([
       [
         { key: "0200", value: "00" },
         { key: "0201", value: "01" },
       ],
-      [{ key: "0202", value: "02" }]
+      [{ key: "0202", value: "02" }],
+    ]);
+  });
+});
+
+describe("PsbtV2.removePartialSig", () => {
+  let psbt;
+
+  beforeEach(() => {
+    psbt = new PsbtV2();
+    psbt.handleSighashType = jest.fn();
+    // This has to be added so inputs can be added else removeSig will fail
+    // since the input at the index does not exist.
+    psbt.PSBT_GLOBAL_TX_MODIFIABLE = ["INPUTS"];
+  });
+
+  it("Throws on validation failures", () => {
+    const removeSig = (index, pub, sig) =>
+      psbt.removePartialSig(index, pub, sig);
```

Comment:
I don't think `sig` is actually passed in either test here. so could just be 
```suggestion
    const removeSig = (index, pub) =>
      psbt.removePartialSig(index, pub);
```